pymeasure/instruments/keithley/keithley2400.py

```diff
@@ -508,7 +509,7 @@ def triad(self, base_frequency, duration):
     )
 
     @property
-    def error(self):
```

Comment:
A `super` call is possible.

I added a test for the keithley2400. According to the manual, the error messages have parenthesis.
This additional processing removes them for neater strings.

Reply:
Actually, looking at the code, it seems to me that removes character `“` from string associated to error code.
It looks like something related to personal taste, on the other hand the ‘retry’ when reply length < 2, it seems important, if really needed.

It would be nice to remove completely this custom next_error to simplify code, but perhaps it is not possible. In any case, I am good to merge.